hooks/pre-checkout

```diff
@@ -62,6 +62,7 @@ export_buildkite_repo() {
   local token
   token=$(jq -r '.body' <"${out_file}" | jq -r '.token')
 
+  export GITHUB_ACCESS_TOKEN="${token}"
```

Comment:
Can this have a more specific name? I think a lot of scripts will have their own `GITHUB_ACCESS_TOKEN` if they use the github API, but this token should only be used to fetch repos.

Reply:
Yes.

Reply:
Done in 0ca2f4ee3c26078b328c4a645991e2496aabdbda